friendModel.js

```diff
@@ -0,0 +1,33 @@
+//Three:
+// import mongoose
+const mongoose = require("mongoose");
+//Create a schema that the database will follos to add friends
+
+const definition =
+{
+    firstName: {
+        type: String, // String, required
+        required: true
+    },
+    lastName: {
+        type: String,  // String, required
+        required: true
+    },
+    age: {
```

Comment:
For this portion consider using a custom validation for the age to be between 1 to 120.
Reference this document: http://mongoosejs.com/docs/validation.html#custom-validators